src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const THREE_DAY_DISCOUNT = 20;
+  const SEVEN_DAY_DISCOUNT = 50;
+
+  switch (true) {
```

Comment:
Prefer `if` instead `switch`, `switch` better use when the number of conditions is large